examples/3-static-resources/README.md

```diff
@@ -11,28 +11,6 @@ to only a few specific files and directories at build time. Specifically:
 1. The full contents of the `src` and `tests` directories
 2. Any `*.rs` and `*.toml` files on the same level as the crate's `Cargo.toml`
 
-So, in a project directory structured like this:
-
-```text
-foo-project
-├── src
-│   ├── lib.rs
-│   └── main.rs
-├── tests
-│   └── my_tests.rs
-├── build.rs
-├── default.nix
-├── Cargo.lock
-├── Cargo.nix
-├── Cargo.toml
-├── README.md
-└── result -> /nix/store/qjz1m7pmm23r7fhgbigb2x5pnwqp925j-crate-foo-project-0.1.0
```

Comment:
The nit I left was only for the `result` line.  The tree itself is helpful

Reply:
I see; my mistake! I can restore the tree without the `result` entry, in that case. Would that work, @antigravityla?

Reply:
Okay, amended the last commit!